lib/apiservers/engine/backends/container.go

```diff
@@ -359,8 +359,7 @@ func (c *Container) ContainerResize(name string, height, width int) error {
 	_, err := client.Interaction.ContainerResize(plResizeParam)
 	if err != nil {
 		if _, isa := err.(*interaction.ContainerResizeNotFound); isa {
-			cache.ContainerCache().DeleteContainer(id)
```

Comment:
See that you did.  Modifying the PR.  Still good to have a different error message than No such container, which is incorrect.